static/js/zamboni/devhub.js

```diff
@@ -602,6 +602,12 @@ function initVersions() {
       date.attr('datetime', note['date']);
       date.attr('title', note['date']);
       clone.find('pre:contains("$comments")')[0].textContent = note['comments'];
+      if (note['attachment_url']) {
```

Comment:
Nit: why not insert the node only if the note has an attachment_url. You already do a bunch of dom modification why not just one insert only if needed.

Another alternative would be to se the href and text content in the template (to empty values if there is no attachment) and then let css decide if the element should be hidden based on the child content of the div?

I almost like that option better as it wouldn't require any javascript at all.

Reply:
> Another alternative would be to se the href and text content in the template (to empty values if there is no attachment) and then let css decide if the element should be hidden based on the child content of the div?

I can see doing that, but I don't know if it would remove the need for the js (see the comment on .html), so I feel it would become more confusing if I were to split the logic between both a js and css file?